Sources/App/DTOs/TodoDTO.swift

```diff
@@ -0,0 +1,17 @@
+import Fluent
+import Vapor
+
+struct TodoDTO: Content {
+    var id: UUID?
+    var title: String?
+    
+    var model: Todo {
```

Comment:
Honestly not a fan of a `var` instead of a `func` named like `toModel()`.
Seems too SwiftUI-y / React-y which doesn't match the general looks of how things are on the Server side, at least in my eyes.

Reply:
In my experience, the hard adherence to the use of `func`s with zero parameters just makes things messier at some call sites. When there are no side effects, in the past the distinction was primarily about properties not being able to be `throws` or `async` - a restriction that no longer applies in present-day Swift.

Reply:
That being said, I'd be open to changing the name to the adverbial form, i.e. `asModel`, I can see that being less confusing.

Reply:
I'm used to functions, I feel like it's a bit clearer because we're transforming the type rather than computing a property. That said, I'm not especially precious about it. Though functions do make it easier as soon as you start wanting to pass in relations etc, which I do a lot